test/test_orm.py

```diff
@@ -226,6 +228,17 @@ def test_transportAction(self):
         milk_object = self.session.scalars(select(pycram.orm.object_designator.Object)).first()
         self.assertEqual(milk_object.pose, result[0].object.pose)
 
+    def test_pickUpAction(self):
+        object_description = object_designator.ObjectDesignatorDescription(names=["milk"])
+        action = PickUpActionPerformable(object_description.resolve(), Arms.LEFT, Grasp.FRONT)
+        with simulated_robot:
+            NavigateActionPerformable(Pose([0.6, 0.4, 0], [0, 0, 0, 1])).perform()
+            action.perform()
```

Comment:
for this to test the object_at_execution behavior you have to move the object two times and check if the object has the correct poses in the database, not the final pose.